packages/core/src/dialects/snowflake/query-generator-typescript.ts

```diff
@@ -16,7 +16,7 @@ import type {
   ShowConstraintsQueryOptions,
 } from '../abstract/query-generator.types';
 
-const CREATE_DATABASE_QUERY_SUPPORTED_OPTIONS = new Set<keyof CreateDatabaseQueryOptions>(['charset', 'collate']);
+const CREATE_DATABASE_QUERY_SUPPORTED_OPTIONS = new Set<keyof CreateDatabaseQueryOptions>([]);
```

Comment:
As no dialect supports charset anymore, you can remove this from CreateDatabaseQueryOptions

Reply:
removed